revscoring/features/revision.py

```diff
@@ -0,0 +1,235 @@
+import re
+from datetime import datetime
+
+from pytz import utc
+from revscoring.languages import is_stopword, stem_word
+
+from . import modifiers
+from ..datasources import revision
+from ..languages import is_badword, is_misspelled, is_stopword, stem_word
+from .feature import Feature
+from .util import (CATEGORY_RE, CITE_RE, IMAGE_RE, INFOBOX_RE, MARKUP_RE,
+                   NUMERIC_RE, SECTION_COMMENT_RE, SYMBOLIC_RE)
+
+
+################################# Time #########################################
+
+def process_day_of_week(revision_metadata):
+    
+    dt = datetime.fromtimestamp(revision_metadata.timestamp.unix(), tz=utc)
+    return dt.weekday()
+
+day_of_week= Feature("revision.day_of_week", process_day_of_week,
+                     returns=int, depends_on=[revision.metadata])
+
+def process_hour_of_day(revision_metadata):
+    
+    dt = datetime.fromtimestamp(revision_metadata.timestamp.unix(), tz=utc)
+    return dt.hour
+
+hour_of_day = Feature("revision.hour_of_day", process_hour_of_day,
+                      returns=int, depends_on=[revision.metadata])
+
+################################ Comment #######################################
+
+def process_has_custom_comment(revision_metadata):
+    
+    if revision_metadata.comment is not None:
+        trimmed_comment = SECTION_COMMENT_RE.sub("", revision_metadata.comment)
```

Comment:
Does this handle multiple sections in a single edit summary? E.g. ""/*test*/ some comment /* another test*/""...